docs/build/guides/conventions/cross-contract.mdx

```diff
@@ -0,0 +1,130 @@
+---
+title: Making cross-contract calls
+description: Call a smart contract from within another smart contract
+---
+
+As with developing software in any language, developing a Soroban smart contract with a rich feature set can be a challenging and time-consuming task. Thankfully, someone else might already have solved part of your issues or build components which could be reused. The open source community is vibrant and Stellar's community does not disappoint.
```

Comment:
```suggestion
As with developing software in any language, developing a Stellar smart contract with a rich feature set can be a challenging and time-consuming task. Thankfully, someone else might already have solved part of your issues or build components which could be reused. The open source community is vibrant and Stellar's community does not disappoint.
```

I believe we're generally working towards using the phrase `Stellar smart contract` now. If you can do a quick find/replace, that'd be awesome!

Reply:
Ok no problem, will do 👍